elf2table/src/lib.rs

```diff
@@ -73,6 +76,11 @@ pub fn parse(elf: &[u8]) -> Result<Option<Table>, anyhow::Error> {
             _ => continue,
         };
 
+        if is_defmt_version(name) {
+            // not a defmt symbol
```

Comment:
Can you copy (part of) the PR description here to explain why this is necessary?